Makefile

```diff
@@ -115,19 +115,46 @@ letsencrypt:
 	&& . ./devops/scripts/load_env.sh ./templates/core/tre.env \
 	&& ./templates/core/terraform/scripts/letsencrypt.sh
 
-tre-stop:
-	echo -e "\n\e[34m»»» 🧩 \e[96mStopping TRE\e[0m..." \
+tre-start:
+	echo -e "\n\e[34m»»» 🧩 \e[96mStarting TRE\e[0m..." \
 	&& . ./devops/scripts/check_dependencies.sh azfirewall \
 	&& . ./devops/scripts/load_env.sh ./templates/core/.env \
 	&& . ./devops/scripts/load_env.sh ./devops/.env \
-	&& ./devops/scripts/control_tre.sh stop
+	&& ./devops/scripts/control_tre.sh start
 
-tre-start:
-	echo -e "\n\e[34m»»» 🧩 \e[96mStarting TRE\e[0m..." \
+tre-stop:
+	echo -e "\n\e[34m»»» 🧩 \e[96mStopping TRE\e[0m..." \
 	&& . ./devops/scripts/check_dependencies.sh azfirewall \
 	&& . ./devops/scripts/load_env.sh ./templates/core/.env \
 	&& . ./devops/scripts/load_env.sh ./devops/.env \
-	&& ./devops/scripts/control_tre.sh start
+	&& ./devops/scripts/control_tre.sh stop
+
+firewall-install:
```

Comment:
I haven't yet included these commands as part of the main deploy. Should we include these services by default?